default.cfg

```diff
@@ -43,6 +43,7 @@ music_volume		8
 parallelism_patch_columns		3
 parallelism_realcolor_tint		8
 scale_melt		true
+semi_translucent_fuzz	true

```

Comment:
If it's true by default, even in truecolor mode, it's not as close as possible to vanilla. It must be deactivate by default. Spectres must look the same in indexed mode like they do in truecolor.